packages/tools/cli/package.json

```diff
@@ -17,15 +17,15 @@
     "build:docs": "yarn build:docs:api",
     "build:docs:api": "yarn build:docs:api:metadata && yarn build:docs:api:markdown",
     "build:docs:api:metadata": "api-extractor run -c ./api-extractor.config.json",
-    "build:docs:api:markdown": "api-documenter markdown --input-folder ./dist/docs/api/metadata --output-folder ./dist/docs/markdown",
-    "build:module": "tsc --declaration false --outDir ./dist/module --module CommonJS",
+    "build:docs:api:markdown": "api-documenter markdown --input-folder ./docs/api/metadata --output-folder ./docs/api/markdown",
+    "build:module": "tsc --declaration false --outDir ./dist/module",
     "build:src": "yarn build:module && yarn build:types",
-    "build:types": "tsc --declaration --declarationMap --declarationDir ./dist/types --emitDeclarationOnly --module CommonJS",
+    "build:types": "tsc --declaration --declarationMap --declarationDir ./dist/types --emitDeclarationOnly",
     "clean": "yarn clean:dist",
     "clean:dist": "rimraf ./dist",
     "test": "yarn test:style && yarn test:syntax && yarn test:integration && yarn test:coverage",
     "test:coverage": "yarn test:integration --coverage --reporters=\"jest-silent-reporter\"",
-    "test:integration": "jest ./test/module/**/*.test.js",
+    "test:integration": "jest",
```

Comment:
Question: How does the jest command now find the required test files?

Reply:
jest, by default, will look for any file that matches a specific test patter (`./**/*.test.*`). I removed the glob pattern from this command so that commands like:

```bash
yarn test:integration ./path/to/specific/test.js
```

would work as expected, and a single file can be tested when debugging. When including the glob pattern, it will always include the globbed tests along with the provided file.